### PR TITLE
ACS-8608: Add bulk-ingester component to alfresco-connector-hxi Helm chart

### PR DESCRIPTION
**Pull request by krdabrowski**
https://hyland.atlassian.net/browse/ACS-8608

### CODE REVIEW

**Review by pmacius — COMMENTED** [review]
After quick look for now: add the postgresql dependency for ci just like in repository:
https://github.com/Alfresco/alfresco-helm-charts/blob/main/charts/alfresco-repository/Chart.yaml#L11

File: charts/alfresco-connector-hxi/Chart.yaml [hunk 1]
```diff
@@ -13,3 +13,8 @@ dependencies:
     repository: https://alfresco.github.io/alfresco-helm-charts/
     tags:
       - ci
+  - name: elasticsearch
+    repository: https://helm.elastic.co
+    version: 7.17.3
+    tags:
+      - ci
```
Comment (gionn) [inline charts/alfresco-connector-hxi/Chart.yaml:20]: postgres, not elasticsearch

anyway not mandatory until we are able to make a working helm install on kind

**Review by gionn — APPROVED** [review]

File: charts/alfresco-connector-hxi/Chart.yaml (hunk 1 above)
Comment (krdabrowski) [reply charts/alfresco-connector-hxi/Chart.yaml]: 🤦‍♂️

Comment (krdabrowski) [reply charts/alfresco-connector-hxi/Chart.yaml]: Fixed

**Pull request merged**